Gems/Pointcloud/Assets/Shaders/Pointclouds/Pointclouds.shader

```diff
@@ -8,8 +8,8 @@
   },
   "GlobalTargetBlendState" : {
       "Enable" : false,
-      "BlendSource" : "Zero",
-      "BlendDest" : "One",
+      "BlendSource" : "AlphaSource",
```

Comment:
Is this change required by this feature?

Reply:
No, I'm not sure how it got there